src/naive/date.rs

```diff
@@ -236,6 +236,49 @@ fn test_date_bounds() {
 }
 
 impl NaiveDate {
+    /// Create a `NaiveDate` of the first day of the given year
+    ///
+    /// This takes an `i16` rather than `i32` to ensure it is not
+    /// out of the allowed range
+    pub fn start_year(year: i16) -> NaiveDate {
```

Comment:
Using an `i16` ties into your work in https://github.com/chronotope/chrono/pull/882. But maybe it is best to accept an `i32` until the decision is made to restrict the range of allowed years?